app/src/components/synths/AddSynth.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <form @submit.prevent="handleSubmit">
+    <fieldset>
+      <label>
+        Synth name: <input v-model="synth.name" required>
+      </label>
+      <br>
+      <label>
+        Image url: <input v-model="synth.image">
+      </label>
+      <br>
+      <label>
+        Polyphonic?
+          Yes:<input type="radio" v-model="synth.polyphonic" v-bind:value="true">
+          No:<input type="radio" v-model="synth.polyphonic" v-bind:value="false">
+      </label>
+      <br>
+      <label>
+        Date produced: <input v-model.number="synth.year" required>
```

Comment:
```suggestion
        Date produced: <input type="number" v-model.number="synth.year" required>
```
I would recommend adding an input type of number so the user has less of a chance of entering a string. Currently, there isn't a type for year, but you can use form validation wizardry to simulate that if you would like to look input changing the input type to date and adding patterns, etc.